tools/runners/run-test-suite.sh

```diff
@@ -112,35 +112,31 @@ do
     if [ "$IS_SNAPSHOT" == true ]
     then
         # Testing snapshot
-
         SNAPSHOT_TEMP=`mktemp snapshot-out.XXXXXXXXXX`
 
-        echo -n "[$tested/$total] ${ENGINE#$ROOT_DIR} $ENGINE_ARGS "
-        echo -n "--save-snapshot-for-global $SNAPSHOT_TEMP ${full_test#$ROOT_DIR}: "
-
+        cmd_line="${ENGINE#$ROOT_DIR} $ENGINE_ARGS --save-snapshot-for-global $SNAPSHOT_TEMP ${full_test#$ROOT_DIR}"
         ( ulimit -t $TIMEOUT; $ENGINE $ENGINE_ARGS --save-snapshot-for-global $SNAPSHOT_TEMP $full_test &> $ENGINE_TEMP )
```

Comment:
Since the command line is in a variable now, I think we should combine these lines. This would improve maintenance.